### ISSUE TITLE
Active tab will not suspend

### ISSUE THREAD

**Issue by JeffH1987** [deanoemcke/thegreatsuspender#1046]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: 7.1.0
 * Browser name & version: Chrome 78.0.3904.70
 * Operating system & version: macOS Catalina 10.15.0

The active tab will not suspend, even if I'm actively using another app. The "Never suspend active tab" is unchecked. This is an issue with sites like Facebook, which actively use background processes that prevent sleep and use too much energy. This issue is defeating the purpose of why I installed this extension: ensuring that no tabs are preventing sleep if I suddenly step away without manually activating screen saver/sleep.

**Comment by CollinChaffin** [comment]
Correct.  This is a misnomer if you look at the issue I previously opened on this (below), I also believed this was the case but with the current code, the active tab IN THE ACTIVE WINDOW will NEVER suspend, unless done manually.  However, active tabs in non-focused windows will in fact suspend.

https://github.com/deanoemcke/thegreatsuspender/issues/762

**Comment by JeffH1987** [comment]
In macOS Catalina, the active tab will not suspend even in non-focused windows.

**Comment by JeffH1987** [comment]
As I said above it is possible to unexpectedly step away from a media rich site like facebook and come back an hour later to see that the computer was awake the whole time! Yes, if text was entered somewhere, suspending could be problematic. But if you were just skimming through the feed, article, whatever, it would be great for some users if the active tab was allowed to automatically suspend.

**Comment by CollinChaffin** [comment]
Oh I 100% agree, and if you look at the older issue I opened on this exact question, my thoughts exactly.  Only NON-ACTIVE windows' "active" tabs suspend, but NEVER the active tab in the active window - which I also believe should be included in the "all" suspension (unless that site/url is explicitly whitelisted).

Since Dean has been MIA since April (on all of Github activity still as of today), I'm guessing we may have to craft a PR/fork at this point to make this happen.
